kolena/dataset/evaluation.py

```diff
@@ -26,31 +26,31 @@
 from kolena._api.v2.model import LoadResultsRequest
 from kolena._api.v2.model import Path
 from kolena._api.v2.model import UploadResultsRequest
-from kolena._experimental.dataset._dataset import _iter_dataset_raw
-from kolena._experimental.dataset._dataset import _to_deserialized_dataframe
-from kolena._experimental.dataset._dataset import _to_serialized_dataframe
-from kolena._experimental.dataset._dataset import load_dataset
-from kolena._experimental.dataset.common import COL_DATAPOINT
-from kolena._experimental.dataset.common import COL_DATAPOINT_ID_OBJECT
-from kolena._experimental.dataset.common import COL_EVAL_CONFIG
-from kolena._experimental.dataset.common import COL_RESULT
-from kolena._experimental.dataset.common import validate_batch_size
-from kolena._experimental.dataset.common import validate_dataframe_ids
 from kolena._utils import krequests_v2 as krequests
 from kolena._utils import log
 from kolena._utils.batched_load import _BatchedLoader
 from kolena._utils.batched_load import init_upload
 from kolena._utils.batched_load import upload_data_frame
 from kolena._utils.consts import BatchSize
 from kolena._utils.state import API_V2
+from kolena.dataset.common import COL_DATAPOINT
+from kolena.dataset.common import COL_DATAPOINT_ID_OBJECT
+from kolena.dataset.common import COL_EVAL_CONFIG
+from kolena.dataset.common import COL_RESULT
+from kolena.dataset.common import validate_batch_size
+from kolena.dataset.common import validate_dataframe_ids
+from kolena.dataset.dataset import _iter_dataset_raw
+from kolena.dataset.dataset import _to_deserialized_dataframe
+from kolena.dataset.dataset import _to_serialized_dataframe
+from kolena.dataset.dataset import load_dataset
 from kolena.errors import IncorrectUsageError
 from kolena.errors import NotFoundError
 
 TYPE_EVALUATION_CONFIG = Optional[Dict[str, Any]]
 TEST_ON_TYPE = Optional[Union[str, List[str]]]
 
 
-def _fetch_dataset(dataset: str) -> pd.DataFrame:
```

Comment:
Not from this PR, but this doesn't look to be used?

Reply:
this is used in an unused function in [shipyard](https://github.com/kolenaIO/shipyard/blob/5bc7515ac9617f4fb0b0f6fd1f9090ecad062bab/galleon/galleon/services/v1/ingest/lib/data_ingestion/_utils/dataset.py#L36), I think having the ability to fetch dataset on demand by name is useful in SDK, I used it in my binary classification example  [PR](https://github.com/kolenaIO/kolena/pull/377/files)

Reply:
Can this be covered by `dataset.py::fetch_dataset`?

Reply:
oh I see, this is a dup then, I will remove this